<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="go_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>88</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>417</width>
       <height>317</height>
      </rect>
     </property>
     <widget class="QLabel" name="image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>5000</width>
        <height>321</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>5000</width>
        <height>321</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>421</width>
      <height>36</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="filepath_input">
       <property name="text">
        <string>/home/jenny/Documents/Code/Qt/test3/score.png</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="enter_button">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>22</y>
      <width>178</width>
      <height>248</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="contrast">
       <property name="text">
        <string>Contrast</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="contrast_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="brightness">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightness_slider">
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="sliderPosition">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="original_image">
       <property name="text">
        <string>Original I&amp;mage</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="binarized_image">
       <property name="text">
        <string>Binari&amp;zed Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="removed_staves">
       <property name="text">
        <string>Remo&amp;ved Staves</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="connected_components">
       <property name="text">
        <string>Connected Components</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_image">
       <property name="text">
        <string>Update Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
